### PR TITLE
Remove deprecated request wrappers

### PR DESCRIPTION
**Pull request by JST5000**
## High Level Overview of Change

Remove requests that were deprecated for the 2.0 release.

### Context of Change

Many request wrappers were just as complicated to use, and their return types were too simplistic in many situations. Instead, we recommend directly calling the request with a `Request` object and `client.request`.

### Type of Change

- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Test Plan

CI Passes

### CODE REVIEW

**Comment by justinr1234** [comment]
@JST5000 please merge latest main branch

**Review by justinr1234 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,14 +17,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Added:
 - Added `submit_and_wait` to sign (if needed), autofill, submit a transaction and wait for its final outcome
 - `submit` and `send_reliable_submission` now accept an optional boolean param `fail_hard` (if `True` halt the submission if it's not immediately validated)
 
+### Changed:
+- Allowed keypairs.sign to take a hex string in addition to bytes
+
 ### Fixed:
 - Refactored `does_account_exist` and `get_balance` to avoid deprecated methods and use `ledger_index` parameter
+- Fixed crashes in the SignerListSet validation
```
Comment (ckniffen) [inline CHANGELOG.md:32]: Were these in previous PRs?

**Review by ckniffen — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JST5000) [reply CHANGELOG.md:32]: Ah, good catch - yeah I think those were from previous PRs.

File: tests/integration/it_utils.py [hunk 2]
```diff
@@ -132,29 +133,38 @@ def submit_transaction(
...
 ) -> Response:
     return await sign_and_submit_async(transaction, wallet, client, check_fee=check_fee)
 
 
+# submits a transaction to the ledger and closes a ledger, synchronously
 def sign_and_reliable_submission(
-    transaction: Transaction, wallet: Wallet, use_json_client: bool = True
+    transaction: Transaction,
+    wallet: Wallet,
+    client: SyncClient = JSON_RPC_CLIENT,
```
[the remark this answers is not in the document]
Comment (JST5000) [reply tests/integration/it_utils.py]: Not part of this PR - I'm not sure whether it should be or not. (Reverted everything which was outside of the deprecated functions being removed)

File: xrpl/asyncio/wallet/wallet_generation.py [hunk 3]
```diff
@@ -13,9 +13,9 @@
 _TEST_FAUCET_URL: Final[str] = "https://faucet.altnet.rippletest.net/accounts"
 _DEV_FAUCET_URL: Final[str] = "https://faucet.devnet.rippletest.net/accounts"
 _AMM_DEV_FAUCET_URL: Final[str] = "https://ammfaucet.devnet.rippletest.net/accounts"
-_HOOKS_V2_TEST_FAUCET_URL: Final[
+_HOOKS_V3_TEST_FAUCET_URL: Final[
     str
-] = "https://hooks-testnet-v2.xrpl-labs.com/accounts"
+] = "https://hooks-testnet-v3.xrpl-labs.com/accounts"
```
Comment (ckniffen) [inline xrpl/asyncio/wallet/wallet_generation.py:18]: These changes were in another PR.  Maybe the branch `xrpl-py-2.0` needs to be rebased with `master`.

File: tests/unit/asyn/wallet/test_wallet_generation.py [hunk 4]
```diff
@@ -43,10 +43,10 @@ def test_get_faucet_wallet_amm_dev(self):
         self.assertEqual(get_faucet_url(json_client_url), expected_faucet)
         self.assertEqual(get_faucet_url(ws_client_url), expected_faucet)
 
-    def test_get_faucet_wallet_hooks_v2_test(self):
```
Comment (ckniffen) [inline tests/unit/asyn/wallet/test_wallet_generation.py:46]: Same. Should be in another PR.

**Comment by justinr1234** [comment]
@JST5000 can you merge main?

**Comment by JST5000** [comment]
Superseded by #581
